src/components/search-form.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import '../styles/search-form.scss';
+
+class SearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleCityChange = this.handleCityChange.bind(this);
+    this.state = {
+      searchText: '',
+    };
+  }
+
+  handleInputChange(e) {
```

Comment:
I try to stay away from single letter variables. At a glance it's unclear if `e` refers to `event` or `error`. Different teams have different standards though.